admin-client/src/components/TaskTable.svelte

```diff
@@ -1,5 +1,4 @@
 <script>
-    import prettyBytes from 'pretty-bytes';
```

Comment:
If we aren't using prettyBytes anywhere else, we could remove it from the package.json file

Reply:
Done 🪄